### ISSUE TITLE
source code

### ISSUE THREAD

**Issue by lyc6749** [fmcp/EndToEndIncrementalLearning#1]
can you provide some source code for the step of getting the first model for each step size and iteration？

**Comment by fmcp** [comment]
Hi, for the first steps (without incremental learning), I used  this code: https://github.com/zhanghang1989/ResNet-Matconvnet

**Comment by lyc6749** [comment]
when I use the learning rate 0.01 (80 epoch), 0.001 (80 epochs), 0.0001 (40 epochs) to train the first imdb of batch20  on cifar，I can not gain the same accuracy as you.

**Comment by fmcp** [comment]
I run 100 epochs using the learning rate commented on the paper (and included in the code for the incremental steps).

Don’t forget to include the L2 loss and random noise to the gradients.
